dashboard/src/components/AppView/AppView.tsx

```diff
@@ -269,6 +266,7 @@ class AppView extends React.Component<IAppViewProps, IAppViewState> {
           break;
         case "Ingress":
           result.ingresses.push(resource);
```

Comment:
Do we need to push the ingress here or we could wait for the proper one is pulled via the reference?

Reply:
We can just remove these.